tests.toml

```diff
@@ -4,5 +4,18 @@ test_format = 1.0
 
 [default]
 
-    test_upgrade_from.3a75b9a45f6ef65e8eef32ff5ba0c8d0b7ae9c8d.name = "0.7.18.2~ynh2"
-    test_upgrade_from.c91bfe0fd8b7eae29c389fd156f956369036308d.name = "0.7.18.2~ynh3"
+    # ------------
+    # Tests to run
+    # ------------
+
+    # -------------------------------
+    # Default args to use for install
+    # -------------------------------
+
+    # -------------------------------
+    # Commits to test upgrade from
+    # -------------------------------
+
+    # test_upgrade_from.3a75b9a45f6ef65e8eef32ff5ba0c8d0b7ae9c8d.name = "0.7.18.2~ynh2"
+    # test_upgrade_from.c91bfe0fd8b7eae29c389fd156f956369036308d.name = "0.7.18.2~ynh3"
```

Comment:
These updates don't work any more?

Reply:
yes, you can look at https://ci-apps.yunohost.org/ci/job/16109